.circleci/config.yml

```diff
@@ -42,18 +42,24 @@ executors:
     machine:
       image: "windows-server-2019-vs2019:stable"
     resource_class: windows.medium
-    shell: powershell.exe -ExecutionPolicy Bypass
+    shell: bash.exe --login -eo pipefail
```

Comment:
Hurrah!!!

I think this also fixes: https://github.com/apollographql/router/issues/1944